frontend/src/containers/LibraryPage/LibraryPage.js

```diff
@@ -1,61 +1,169 @@
 import { Grid } from '@material-ui/core';
 import React from 'react';
-import { UploadIcon } from '../../assets';
+import { UploadIcon, emptyLibraryDrawing, McRideScreaming } from '../../assets';
 import EmptyLibraryContent from '../../components/EmptyLibraryContent/EmptyLibraryContent';
 import TopBar from '../../components/TopBar/TopBar';
 import UploadModal from '../../components/UploadModal/UploadModal';
 import './styles.css';
+import VideoCardsByDate from '../../components/VideoCardsByDate/VideoCardsByDate';
+
+const mockedCards = [
```

Comment:
Not sure if this is where you want to keep the mocked data, maybe move it to the constants. Either way, it's temporary, so not that important

Reply:
Yeah, I think it doesn't matter